cmd_init.go

```diff
@@ -171,6 +180,12 @@ func PopulateSchemaDir(s *tengo.Schema, parentDir *Dir, makeSubdir bool) error {
 	if s.Name == parentDir.Config.Get("temp-schema") {
 		return nil
 	}
+	ignoreSchemaRegex := parentDir.Config.Get("ignore-schema-regex")
+	schemaRE, _ := regexp.Compile(ignoreSchemaRegex)
```

Comment:
General `golang` advice : you are ignoring the error value returned from `Compile`. This means the user may enter an invalid regex and the app will be silent about it. The advice is to never ignore an `error`. Add something like:
```
schemaRE, err := regexp.Compile(ignoreSchemaRegex)
if err != nil {
  return fmt.Errorf("Invalid regular expression on ignore-schema-regex: %s; %s", ignoreSchemaRegex, err)
}
```